### DISCUSSION TITLE
`Background` vs. `BackgroundColor`

### DISCUSSION THREAD

**Discussion by AndreasReitberger** [dotnet/maui#17296] [General]
I have issues with the `Button` control that the `BackgroundColor` is set to the bounded value.
However when I use `Background` instead, it works.

So I'm asking my self, what is the right approach to set the background color of a control at all?
Should I switch to `Background` or is just the `BackgroundColor` property buggy?

Thanks in advanced :)

P.S. I already checked the difference here:
https://stackoverflow.com/questions/74034216/net-maui-background-vs-backgroundcolor-what-is-the-difference

**Comment by gabsamples6** [comment]
Hi many bugs have been raised on the issues and in the end I think background is what is actually supported .it’s a messy story and I cannot find the link with explanation but in net 7 you could do same for button … believe fixed in net 8
.ConfigureMauiHandlers(_ => 
{
	LabelHandler.Mapper.AppendToMapping(
	(View.BackgroundColor),
		(handler, View) => handler.UpdateValue(nameof(IView.Background)));
})
